### PR TITLE
Changing navbar to work without javascript.

### PR DESCRIPTION
**Pull request by 2-B**
This commit fix issue [#24](https://github.com/ccc-ch/ccc-ch.ch/issues/24).

### CODE REVIEW

File: theme/static/bootstrap/navbar.css [hunk 1]
```diff
@@ -7,6 +7,49 @@ body {
   margin-bottom: 20px;
 }
 
+/*
+ * Open dropdowns on hover instead of click.
```
Comment (dbrgn) [inline theme/static/bootstrap/navbar.css]: Does that work on touch devices?

Comment (2-B) [inline theme/static/bootstrap/navbar.css]: I didn't test it. But good question.

**Comment by 2-B** [comment]
It didn't work properly. Now it should work, but touch devices still need javascript.

**Comment by bashvi** [comment]
I think that should do it. The probability of someone having a touch device and no Javascript is probably rather small. And even if - this is still an improvement over the old situation.

**Comment by dbrgn** [comment]
Yeah, works nicely :)

**Comment by dbrgn** [comment]
Deployed.

**Pull request merged**